### PR TITLE
Update Primer hook priorities 

### PR DESCRIPTION
**Pull request by EvanHerman**
The new version of primer add's priorities to some functions to allow for more precise injection of content or easier management of the content flow.

This patch add's the required priorities to ensure compatibility with the latest version of primer.

**Update:** Additional tweaks for WooCommerce styles. For a complete list see https://github.com/godaddy/wp-primer-theme/pull/182

**Requires:** https://github.com/godaddy/wp-primer-theme/pull/146

### CODE REVIEW

File: editor-style-rtl.css [hunk 1]
```diff
@@ -1882,7 +1882,7 @@ input[type="submit"] {
   input[type="button"]:hover,
   input[type="reset"]:hover,
   input[type="submit"]:hover {
-    background: transparent !important; }
+    background: transparent; }
```
Comment (frankiejarrett) [inline editor-style-rtl.css]: @EvanHerman This causes a bug with all submit buttons on the site, regardless of if WooCommerce is in use.

screen shot 2017-02-15 at 2 15 23 pm

**Comment by EvanHerman** [comment]
@fjarrett I think the issue was on the :hover/:focus states a few lines below that. I went ahead and tweaked the styles, however I'm not sure how I feel about the solution because I know we're trying to keep WooCommerce and default styles separate.

One additional bug I did catch was the theme was appending arrows onto the direct manipulation buttons in the customizer using the :after pseudo class.

**Comment by EvanHerman** [comment]
The latest commit fixes the WooCommerce add to cart button overlap/overflow on i10n sites.

I also went ahead and added pagination styles to this branch and closed the other related PR.

example

**Comment by EvanHerman** [comment]
~~There are a few other tweaks that must be made before this can be merged. The add to cart buttons and variations buttons all go transparent when hovered on. The WooCommerce disabled variation button is still purple.~~

**Pull request merged**